cmd/pctl/add.go

```diff
@@ -24,6 +24,13 @@ var createPRFlags = []cli.Flag{
 		Value: false,
 		Usage: "If given, upgrade will create a PR for the modifications it outputs.",
 	},
+	&cli.StringFlag{
+		Name:        "pr-message",
+		Value:       "Push changes to remote",
+		DefaultText: "Push changes to remote",
```

Comment:
what difference between `value` and `defaulttext`?

Reply:
Default text is displayed in the help and value is the actual value.